generator/generator-opc-generator/generators/app/index.js

```diff
@@ -0,0 +1,62 @@
+//@ts-nocheck
+'use strict';
+const Generator = require('yeoman-generator');
+const chalk = require('chalk');
+const yosay = require('yosay');
+const fs = require( 'fs' );
+
+module.exports = class extends Generator {
+  prompting() {
+    // Have Yeoman greet the user.
+    this.log(
+      yosay(`Welcome to the super ${chalk.red('opc-generator')} generator!`)
+    );
+
+    const prompts = [
+      {
+        type: 'input',
+        name: 'elementName',
+        message: 'What would you like to call your element? ',
+        default: 'my-element'
+      },
+      {
+        type: 'input',
```

Comment:
Lets pick up the authorname and email from git config. We have done this for one platform microservice generator too

Reply:
Added prompts to get author & email